types/responsepolicy.pp

```diff
@@ -0,0 +1,12 @@
+type Dns::ResponsePolicy = Array[
+  Struct[{
+    zone           => String[1],
+    policy         => Optional[Enum[
+      'given', 'disabled', 'passthru', 'drop',
+      'nxdomain', 'nodata', 'tcp-only', 'cname'
+    ]],
+    cname_domain   => Optional[String[1]],
+    max_policy_ttl => Optional[Integer],
```

Comment:
Can we tighten this with a lower bound like `0`?
```suggestion
    max_policy_ttl => Optional[Integer[0]],
```

Reply:
okay, no problem